dll/win32/imm32/imm.c

```diff
@@ -2609,23 +2609,29 @@ BOOL WINAPI ImmIsUIMessageW(
 BOOL WINAPI ImmNotifyIME(
   HIMC hIMC, DWORD dwAction, DWORD dwIndex, DWORD dwValue)
 {
-    InputContextData *data = get_imc_data(hIMC);
+    DWORD idImeThread, idThread;
```

Comment:
usual win32 code practice uses `dwThreadId` (or just `dwThread` sometimes) . For example: `dwImeThreadId` etc.

Reply:
Renamed. OK.